### ISSUE TITLE
Add missing sensors ID_WEB_HUPout, ID_WEB_ZW1out, ID_WEB_Zaehler_BetrZeitZWE2

### ISSUE THREAD

**Issue by pamo85** [BenPru/luxtronik#239]
I would like to have following sensors integrated

# 39
- group: calculations
  id: ID_WEB_HUPout
  friendly_name: LUX2 Ausgang 'Heizungsumwälzpumpe'
  icon: mdi:heat-pump-outline

# 48
- group: calculations
  id: ID_WEB_ZW1out
  friendly_name: LUX2 Ausgang 'Steuersignal Zusatzheizung v. Heizung'
  icon: mdi:heat-pump-outline

# 61
- group: calculations
  id: ID_WEB_Zaehler_BetrZeitZWE2
  friendly_name: LUX2 Betriebsstunden Zweiter Wärmeerzeuger 2
  icon: mdi:history

thx for checking it

**Comment by BenPru** [comment]
> # 39 id: ID_WEB_HUPout

see binary_sensor.luxtronik*_circulation_pump_heating

> # 48 id: ID_WEB_ZW1out

see binary_sensor.luxtronik*_additional_heat_generator

> # 61 id: ID_WEB_Zaehler_BetrZeitZWE2

Really not exists.

**Comment by pamo85** [comment]
My fault. Thx.  I tried to find it with developer tools. but as only activated entities are shown I couldn't find it.

So just one missing :-)

**Comment by Kars-de-Jong** [comment]
> 
> > # 61 id: ID_WEB_Zaehler_BetrZeitZWE2
> 
> Really not exists.

What do you mean? It does exist, it is listed in `calculations.py`, and can be read:
```
Number: 61    Name: ID_WEB_Zaehler_BetrZeitZWE2                                  Type: Seconds              Value: 0
```

**Comment by pamo85** [comment]
> > > # 61 id: ID_WEB_Zaehler_BetrZeitZWE2
> > 
> > 
> > Really not exists.
> 
> What do you mean? It does exist, it is listed in `calculations.py`, and can be read:
> 
> ```
> ...
> Number: 78    Name: ID_WEB_Code_WP_akt                                           Type: HeatpumpCode         Value: MSW2-6S
> ...
> Number: 61    Name: ID_WEB_Zaehler_BetrZeitZWE2                                  Type: Seconds              Value: 0
> ```

Yes it's available and I can use it as additional sensor with bouni integration. but it's not integrated as sensor in BenPru integration.

**Comment by rhammen** [comment]
ID_WEB_Zaehler_BetrZeitZWE2   is available as 'additional_heat_generator2_operation_hours' in 2025.07.22-Beta
So all requested sensors are now (already) part of the integration.

It looks like 'additional_heat_generator2_operation_hours' has a problem with the translations, so its friendly name may be only 'HP' in english. This will be fixed in an upcoming release with updated translations.

Since all sensors are available, issue can now be closed as completed.
